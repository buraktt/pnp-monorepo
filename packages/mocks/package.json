{
  "name": "@monorepo/mocks",
  "version": "0.0.1"
}
