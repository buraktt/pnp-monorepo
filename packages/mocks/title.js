export const titleMocks = {
  title: 'Title test',
  color: 'primary'
}
