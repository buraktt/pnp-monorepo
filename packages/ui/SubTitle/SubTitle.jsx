import React from 'react'
import { Typography } from '@material-ui/core'

export default function SubTitle(props) {
  const { title, color } = props
  return (
    <Typography variant="h2" color={color}>
      {title}
    </Typography>
  )
}
