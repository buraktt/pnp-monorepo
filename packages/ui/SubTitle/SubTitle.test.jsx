/**
 * @group snapshot
 */

import React from 'react'
import SubTitle from './SubTitle'
import { render } from '@testing-library/react'
import { titleMocks } from '@monorepo/mocks/title'

describe('<SubTitle />', () => {
  it('should render without errors', () => {
    const { asFragment } = render(<SubTitle {...titleMocks} />)
    expect(asFragment()).toMatchSnapshot()
  })
})
