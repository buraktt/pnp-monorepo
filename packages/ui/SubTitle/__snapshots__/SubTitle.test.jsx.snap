// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SubTitle /> should render without errors 1`] = `
<DocumentFragment>
  <h2
    class="MuiTypography-root MuiTypography-h2 MuiTypography-colorPrimary"
  >
    Title test
  </h2>
</DocumentFragment>
`;
